docs/LICENSE

```diff
@@ -0,0 +1,13 @@
+Copyright 2008-2013 Concur Technologies, Inc.
```

Comment:
This is a source header and not a license file.

Are Concur the author of Slate? ie: It's injecting its license into its output?

Reply:
I believe they are. I'm new to open source -- is it ok to remove the docs/LICENSE file? Almost everything inside docs/* is cloned from the slate github repo.